### PR TITLE
Updates render function

### PR DESCRIPTION
**Pull request by trevoruptain**

### CODE REVIEW

File: react-minesweeper/components/board.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Board extends React.Component{
       <div id="board">
         <div>WHATS UP!!!!!!!!!!!!!!!!!</div>
         <div>WHAT IS LEFT?!!!!!!!!!!!!!!!!!</div>
-
+        <div>What is right???</div>
```
Comment (joshling1919) [inline react-minesweeper/components/board.jsx]: let's change this to 'What is down?' instead

Comment (trevoruptain) [reply react-minesweeper/components/board.jsx]: @joshling1919 wording has been updated

**Comment by trevoruptain** [comment]
@joshling1919 wording has been updated

**Comment by joshling1919** [comment]
@trevoruptain looks good to me! merging into master now.

**Pull request merged**